microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -747,20 +747,38 @@ To enable diagnostic data collection, go to `edge://settings/privacy`. Turn on t
 <!-- ====================================================================== -->
 ## Writing assistance
 
-Microsoft Edge provides enhanced spelling and grammar checking as you type in the browser.  To use enhanced spelling and grammar checking, go to `edge://settings/languages` and select **Microsoft Editor** under the **Use writing assistance** toggle. 
+To help you write faster and with fewer mistakes on the web, Microsoft Edge provides writing assistance tools, including enhanced spelling and grammar checking and text prediction. 
```

Comment:
```suggestion
To help you write faster and with fewer mistakes on the web, Microsoft Edge provides writing assistance tools, including spell checking, grammar checking, and text prediction. 
```
heading toward making this a more general intro to the section; only 1 of the 3 options is 'enhanced'.

avoiding "x and y and z" construction